### PR TITLE
crystal-icr 0.2.13 (new formula)

### PR DESCRIPTION
**Pull request by damireh**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests]
(https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Interactive console for Crystal programming language

*Trying to revive what was started here https://github.com/Homebrew/homebrew-core/pull/1579 and https://github.com/Homebrew/homebrew-core/pull/2526

### CODE REVIEW

File: Formula/crystal-icr.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  depends_on "crystal-lang"
+  depends_on "readline"
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    assert_equal "icr version #{version}\n", shell_output("#{bin/"icr"} -v").lines.first
```
Comment (MikeMcQuaid) [inline Formula/crystal-icr.rb]: Use `assert_match` and skip the `lines.first`.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
One nit otherwise 👍

File: Formula/crystal-icr.rb (hunk 1 above)
Comment (damireh) [inline Formula/crystal-icr.rb]: done

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**